packages/admin/test/e2e/cli.feature.ts

```diff
@@ -55,8 +55,6 @@ describe('CLI', () => {
   beforeAll(() => {
     jest.setTimeout(45_000);
 
-    expect(existsSync(`${CLI_EXECUTABLE}`)).toBe(true);
```

Comment:
Seems like `expect()` outside of an `it` or `test` block don't get executed: https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-standalone-expect.md#rule-details

Reply:
Then if we want to keep this check we would need to replace it with an if statement followed by a throw, correct?

Reply:
I just saw you put it in a separate `it` block which makes even more sense 🔥